src/main/java/org/olf/folio/order/listeners/ConfigurationListener.java

```diff
@@ -27,10 +27,11 @@ public void contextInitialized(ServletContextEvent sce) {
 		 
 				config.setProperty("permELocation", System.getenv("permELocation")); 
 				config.setProperty("fiscalYearCode", System.getenv("fiscalYearCode"));
-				config.setProperty("loanType", System.getenv("loanType"));
-				config.setProperty("textForElectronicResources", System.getenv("textForElectronicResources"));
+				// config.setProperty("loanType", System.getenv("loanType"));
+				// config.setProperty("textForElectronicResources", System.getenv("textForElectronicResources"));
 				config.setProperty("noteType", System.getenv("noteType"));
 				config.setProperty("materialType", System.getenv("materialType"));
+				config.setProperty("billTo", System.getenv("billTo"));
```

Comment:
We should add `billTo` to [.env.example](https://github.com/cul-it/order-import-poc/blob/add-billTo/.env.example) and document it in [Docker.md](https://github.com/cul-it/order-import-poc/blob/add-billTo/Docker.md).

Reply:
I don't know how to or if I can modify the .env file stored in S3.  The Docker.md file doesn't explicitly mention all the env files and just has: "Edit other properties as needed"...is that not sufficient?

Reply:
That's probably sufficient for now, especially if we're providing a usable example value for Cornell. This highlights that we need to revisit overall documentation (README/Docker/Deployment) down the road.